qutebrowser/misc/sql.py

```diff
@@ -339,77 +470,83 @@ class SqlTable(QObject):
 
     changed = pyqtSignal()
 
-    def __init__(self, name, fields, constraints=None, parent=None):
+    def __init__(self, database: Database, name: str, fields: List[str], constraints:
+                 Optional[Dict[str, str]] = None, parent: Optional[QObject] = None):
```

Comment:
```suggestion
                 Optional[Dict[str, str]] = None, parent: Optional[QObject] = None) -> None:
```

Reply:
Thanks. I thought `__init__`'s return type was not annotated. I fixed a couple more.